src/module.c

```diff
@@ -3348,29 +3349,31 @@ robj **moduleCreateArgvFromUserFormat(const char *cmdname, const char *fmt, int
  * * **cmdname**: The Redis command to call.
  * * **fmt**: A format specifier string for the command's arguments. Each
  *   of the arguments should be specified by a valid type specification:
- *   b    The argument is a buffer and is immediately followed by another
+ *
+ *   * b: The argument is a buffer and is immediately followed by another
  *        argument that is the buffer's length.
- *   c    The argument is a pointer to a plain C string (null-terminated).
- *   l    The argument is long long integer.
- *   s    The argument is a RedisModuleString.
- *   v    The argument(s) is a vector of RedisModuleString.
+ *   * c: The argument is a pointer to a plain C string (null-terminated).
+ *   * l: The argument is long long integer.
+ *   * s: The argument is a RedisModuleString.
+ *   * v: The argument(s) is a vector of RedisModuleString.
  *
  *   The format specifier can also include modifiers:
- *   !    Sends the Redis command and its arguments to replicas and AOF.
- *   A    Suppress AOF propagation, send only to replicas (requires `!`).
- *   R    Suppress replicas propagation, send only to AOF (requires `!`).
+ *
+ *   * !: Sends the Redis command and its arguments to replicas and AOF.
+ *   * A: Suppress AOF propagation, send only to replicas (requires `!`).
+ *   * R: Suppress replicas propagation, send only to AOF (requires `!`).
  * * **...**: The actual arguments to the Redis command.
```

Comment:
looks like this one didn't get indented correctly.

Reply:
This is correct actually. It's a nested list.

Reply:
Ah, I see redis.io isn't rendered the same way as Github. In Github it looks right: https://github.com/redis/redis-doc/blob/master/topics/modules-api-ref.md#redismodule_call

Reply:
I don't know which markdown renderer is used in redis.io, but according to the original markdown spec (https://daringfireball.net/projects/markdown/syntax#list) the body of multi-paragraph lists should be indented 4 spaces, so I'll do that and hope it will work.